.gitignore

```diff
@@ -8,9 +8,11 @@ node_modules/
 # hardhat
 artifacts
 cache
-deployments/localhost
-deployments/hardhat
+/deployments
```

Comment:
This will match the absolute path `/deployments`, no? Which is probably not what we want.

Reply:
Will we be committing our testnet or mainnet deployment files?

Reply:
I think that's the convention with hardhat projects, no? You want to keep around the `deployments/network` directory as it keeps states that's necessary for future deployments to run, right?

Reply:
Makes sense, specially that we are using upgradable contracts. I made the changes to explicitly ignored the local networks only.